config/application.rb

```diff
@@ -45,14 +53,16 @@ def config
   class Application < Rails::Application
 
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 6.0
+    config.autoloader = :classic
```

Comment:
`Zeitwerk` is better here because it address these [problems](https://guides.rubyonrails.org/v5.2/autoloading_and_reloading_constants.html#common-gotchas)

Reply:
Zeitwerk is coming with https://github.com/SUSE/rmt/tree/zeitwerk, but it's a large change and requires https://github.com/SUSE/rmt/pull/616 to be merged first.

Reply:
If you're willing to review all of those changes as well, I can add them to this PR ^_^

Reply:
@ngetahun I added a new commit with `zeitwerk`